src/cryptoadvance/specter/services/extension_gen.py

```diff
@@ -46,13 +48,20 @@ def __init__(
         self.base_path = base_path
         self.org = ext_org
         self.id = ext_id
+        self.encrypted_userdata = encrypted_userdata
+        self.custom_version = custom_version
         self.isolated_client = isolated_client
         self.devicename = devicename
         self.author = author
         self.author_email = email
         vc = VersionChecker()
         version = vc._get_current_version()
-        if version == "custom":
+        if self.custom_version != "":
+            if vc._check_if_version_is_available(custom_version) == True:
```

Comment:
`== True` is not needed. In order to improve readability, you could rename the method and call it:
```suggestion
            if vc._version_is_available(custom_version):
```
Comparing methods which return booleans explicitely is a common issue. It's not wrong but i hope the readability improvement above makes sense to you.